packages/cli/scripts/deploy.ts

```diff
@@ -35,8 +35,8 @@ export async function deployAndSaveProgress<Settings = unknown>(
     scriptExecuted = await deploy(configuration, networkId, deployments, fromIndex, toIndex, silent)
   } catch (error) {
     await deployments.saveToFile(deploymentsFile, configuration, false)
-    console.error(`Failed to deploy the project`)
-    throw error
+    console.error(`Failed to deploy the project, error: `, error)
```

Comment:
Let's only log this when debug mod is enabled, since we throw error with stack trace now.